tests/ixia/test_ixia_traffic_restpy.py

```diff
@@ -0,0 +1,198 @@
+###############################################################################
+# This test cases demonstrates: 
+#   * All the fixtures required for running ixia script (please see the 
+#     arguments of the test function)
+#   * How Ixia chassis card/ports are addressed
+#   * How you can configure/control ixia devices, start traffic and collect 
+#     statistics using REST API
+#   * This simple sanity test cases can be used to check if testbed setup
+#     is correct or not - since it prints a lot of testbed data
+###############################################################################
+
+import logging
+import time
+import pytest
+import pprint
+from common.utilities import wait_until
+from common.fixtures.conn_graph_facts import conn_graph_facts
+from common.platform.interface_utils import check_interface_information
+from common.platform.daemon_utils import check_pmon_daemon_status
+from common.reboot import *
+from common.platform.device_utils import fanout_switch_port_lookup
+from common.helpers import assertions
+from ixnetwork_restpy import SessionAssistant, Files
```

Comment:
Can you move all the low level IXIA API functions to ixia_helpers.py?

Reply:
Thanks for the suggestion. In fact we are in process of designing those high-level wrapper function. See you soon in PRs and meeting and may be design discussion regarding this. Currently I suggest let it be there, to get the ball rolling. As soon we will formulate the new wrapper API we will remove these and replace these with new wrapper functions (which may be bit delayed).  Till then let it be here

Reply:
I have moved some of them to the ixia_fixture.py (mainly connection related). Rest of them is still there. Until we come up with some high level wrapper they suppose to be there. This is only a sample script. This will act as a templet for the engineers who will add test-case to act with ixia devices.
Purpose is the get the activity started. Later we may enhance/change or even remove it (after a considerable RDMA test case is added). Right now to stat the activity it is important.

Reply:
I have also moved the ixia libraries to tests/ixia/lib/* and changed the import statements on the top of the script accordingly.

Reply:
I have also moved the ixia libraries to "/sonic-mgmt/tests/common/ixia" and changed the import statements on the top of the script accordingly.